package.json

```diff
@@ -69,4 +70,4 @@
     "uppercase",
     "vscode"
   ]
-}
+}
```

Comment:
Keep the newline at end of file. Avoid doing it in a new commit, just amend the existing one.